### PR TITLE
Add code to find emails on the PA contact-your-rep page

### PR DESCRIPTION
**Pull request by samtregar**
Looks like this only gets us 37 email addresses, all senate, but it's better than none.

### CODE REVIEW

File: openstates/pa/people.py [hunk 1]
```diff
@@ -56,11 +63,21 @@ def scrape_photo_url(self, url, page, person):
         else:
             raise AssertionError("Legislator photo parsing needs to be rewritten")
 
-    def scrape_email_address(self, url, page):
-        if re.search(r'var \S+\s+= "(\S+)";', page):
-            vals = re.findall(r'var \S+\s+= "(\S+)";', page)
-            return '%s@%s%s' % tuple(vals)
+    def scrape_email_address(self, page, leg_id):
+        if leg_id == "1011":
```
Comment (jamesturk) [inline openstates/pa/people.py]: let's make sure to remove this

**Comment by jamesturk** [comment]
looks pretty good, mind fixing the flake8 errors and removing the ipdb, then 👍

File: openstates/pa/people.py (hunk 1 above)
Comment (samtregar) [inline openstates/pa/people.py]: Ha, whoops!  Will do.

**Comment by samtregar** [comment]
Dropped the debug code (gah) and fixed the flake8 warnings.

**Pull request merged**